sdk/program/src/sanitize.rs

```diff
@@ -10,13 +12,15 @@ pub enum SanitizeError {
     InvalidValue,
 }
 
-/// Trait for sanitizing values and members of over the wire messages.
-/// Implementation should recursively decent through the data structure
-/// and sanitize all struct members and enum clauses.  Sanitize excludes
-/// signature verification checks, those are handled by another pass.
-/// Sanitize checks should include but are not limited too:
-///   * All index values are in range
-///   * All values are within their static max/min bounds
+/// A trait for sanitizing values and members of over the wire messages.
+///
+/// Implementation should recursively decent through the data structure and
+/// sanitize all struct members and enum clauses. Sanitize excludes signature
+/// verification checks, those are handled by another pass. Sanitize checks
+/// should include but are not limited to:
```

Comment:
```suggestion
/// A trait for sanitizing values and members of over-the-wire messages.
///
/// Implementation should recursively descend through the data structure and
/// sanitize all struct members and enum clauses. Sanitize excludes signature-
/// verification checks, those are handled by another pass. Sanitize checks
/// should include but are not limited to:
```
(predates your changes)